frontend/src/employee-frontend/components/vasu/VasuEditPage.tsx

```diff
@@ -151,12 +156,17 @@ export default React.memo(function VasuEditPage({
             {showSpinner && <Spinner />}
           </StatusContainer>
           {vasu && (
-            <Button
-              text={i18n.vasu.checkInPreview}
-              disabled={status.state != 'clean'}
-              onClick={() => history.push(`/vasu/${vasu.id}`)}
-              primary
-            />
+            <FullWidthDiv>
```

Comment:
The button position flickers when StatusContainer changes (textual changes and spinner). Oh well, if this is what the UX department wanted, then let's roll with it 😃